lib/travis/build/addons/deploy/script.rb

```diff
@@ -311,6 +323,22 @@ def owner_name
             def compact(hash)
               hash.reject { |_, value| value.nil? }.to_h
             end
+
+            def want_pre_19?
+              @want_pre_19
+            end
+
+            def dpl_version
+              config[:dpl_version].nil? ? nil : config[:dpl_version].to_s.shellescape
+            end
+
+            def dpl_deprecation_message
+              ENV['DPL_DEPRECATE_MESSAGE']
+            end
+
+            def dpl_ruby_version
+              dpl2? || want_pre_19? ? '$(travis_internal_ruby)' : '2.7'
```

Comment:
Is it possible to revert this back to always use `$(travis_internal_ruby)`, rather than hardcoding `2.7` for some cases?

As-is this prevents us from using `dpl` v1.10.16 (ruby-2.7 doesn't exist as an `rvm` option on trusty).